scripts/index.js

```diff
@@ -167,6 +171,22 @@ function copyPublicFolder() {
   });
 }
 
+function runHook(hook) {
+  if (!hook || typeof hook !== 'string') {
+    return;
+  }
+
+  const exec = require('child_process').exec;
+
+  const child = exec(hook, (error, stdout, stderr) => {
+    console.log(`${stdout}`);
```

Comment:
I don't think the template string is necessary here since stdout and stderr are already strings and you don't add anything to the string themselves.